resources/js/Components/Forms/Dropdown.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+interface DropdownProps {
```

Comment:
This needs a callback passed through, that is used when the selected value is changed by user input